wiki/en/en-Installation-for-Windows.md

```diff
@@ -9,30 +9,28 @@ permalink: "/wiki/Installation-for-Windows"
 Make sure you read the [Getting Started](Getting-Started) page.
 1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install [this free ASIO driver (ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} before installing Jamulus.
 1. [Download Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: .button target="_blank" rel="noopener noreferrer"}
-1. **Install Jamulus**. Execute the Jamulus installer. If you get a warning notice from SmartScreen, click on "More info" and "Run anyway" to install Jamulus. This is needed since we don't pay for code signing yet.
+1. **Install Jamulus**: Execute the Jamulus installer. If you get a warning notice from SmartScreen, click on "More info" and "Run anyway" to install Jamulus. This is needed since we don't pay for code signing yet.
```

Comment:
"Execute" is a bit of a technical word :-) Stylistically, perhaps "run" or even "double click" would be better here. Also, what's SmartScreen? Isn't it just a warning dilogue?

Reply:
I think double click would be better. And yes, the dialog is from SmartScreen

Reply:
Now the dialog from SmartScreen is gone (probably since Jamulus was downloaded by a specific number of Windows users which now adds this version to a white list?)